### PR TITLE
Add possibility to comment questions

### PR DESCRIPTION
**Pull request by ElaPl**
Comment can be add only if question is not active.
Newest comments are at the top of the l
I added button to "Dodaj komentarz" besides comment form on question side. It looks better.

### CODE REVIEW

File: izp/polls/models.py [hunk 1]
```diff
@@ -155,3 +156,19 @@ class Vote(models.Model):
     def __str__(self):
         return self.question.question_text + ' ' + \
             self.choice.choice_text + ' ' + str(self.code)
+
+
+class Comment(models.Model):
+    question = models.ForeignKey(Question, on_delete=models.CASCADE,
+                                 related_name='comments')
+    context = models.TextField()
```
Comment (czapiga) [inline izp/polls/models.py]: context? I'd rather name this variable `text`. Please specify max length of comment.

File: izp/polls/models.py [hunk 2]
```diff
@@ -155,3 +156,19 @@ class Vote(models.Model):
...
+    created_date = models.DateTimeField(default=timezone.now)
```
Comment (czapiga) [inline izp/polls/models.py]: created_date -> date

File: izp/polls/templates/polls/comment_form.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{% extends 'polls/base.html' %}
+
+{% block content %}
+    <h1>Nowy komentarz</h1>
+    <form method="POST" class="post-form">{% csrf_token %}
+        {{ form.as_p }}
+        <button type="submit" class="save btn btn-default">Dodaj komentarz</button>
+    </form>
+{% endblock %}
```
Comment (czapiga) [inline izp/polls/templates/polls/comment_form.html]: Please add new line at the end of file

File: izp/polls/views.py [hunk 4]
```diff
@@ -282,3 +288,22 @@ def deactivate_question(request, question_id):
 
     return HttpResponseRedirect(reverse('polls:poll_detail',
                                         args=(question.poll.id,)))
+
+
+def add_comment_to_question(request, question_id):
+    question = get_object_or_404(Question, pk=question_id)
+    if question.is_active():
```
Comment (czapiga) [inline izp/polls/views.py]: This check is invalid - I think you want to check if question wasn't started.

**Comment by czapiga** [comment]
1. Please move comment input box to question detail view (I thought we talked about it).
2. Please inform user if it's impossible to add comment. Now you're just ignoring fact that user clicked on button and it feels like that button is not working.
3. I think you should disable possibility of resizing comment input field.
4. Please rename form fields names in view.

File: izp/polls/views.py (hunk 4 above)
Comment (ElaPl) [reply izp/polls/views.py]: Changed

File: izp/polls/templates/polls/comment_form.html (hunk 3 above)
Comment (ElaPl) [reply izp/polls/templates/polls/comment_form.html]: file deleted

File: izp/polls/models.py (hunk 2 above)
Comment (ElaPl) [reply izp/polls/models.py]: Changed

File: izp/polls/models.py (hunk 1 above)
Comment (ElaPl) [reply izp/polls/models.py]: Changed

File: izp/polls/models.py [hunk 5]
```diff
@@ -155,3 +156,25 @@ class Vote(models.Model):
...
+
+
+class Comment(models.Model):
+    question = models.ForeignKey(Question, on_delete=models.CASCADE,
+                                 related_name='comments')
+    text = models.TextField()
+    date = models.DateTimeField(default=timezone.now)
+
+    def __str__(self):
+        return self.context
```
Comment (czapiga) [inline izp/polls/models.py]: ?????????????

File: izp/polls/models.py [hunk 6]
```diff
@@ -155,3 +156,25 @@ class Vote(models.Model):
     def __str__(self):
         return self.question.question_text + ' ' + \
             self.choice.choice_text + ' ' + str(self.code)
+
+
+class Comment(models.Model):
+    question = models.ForeignKey(Question, on_delete=models.CASCADE,
+                                 related_name='comments')
+    text = models.TextField()
```
Comment (czapiga) [inline izp/polls/models.py]: Please specify limit for `text` field

**Comment by czapiga** [comment]
Please add tests

File: izp/polls/models.py (hunk 6 above)
Comment (ElaPl) [reply izp/polls/models.py]: There is limit 500 words.
Look at CommentForm:
        self.fields['text'].widget.attrs['maxlength'] = '500'

File: izp/polls/views.py [hunk 7]
```diff
@@ -1,10 +1,13 @@
-from django.shortcuts import get_object_or_404, render
+from django.shortcuts import get_object_or_404, render, redirect
```
Comment (czapiga) [inline izp/polls/views.py]: Check if new import is really needed

**Pull request merged**